tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -75,9 +75,17 @@ public void Decompress(Util.CompressionType type)
             byte[] data = File.ReadAllBytes(testFilePath);
             var bytes = new byte[bufferSize];
             foreach (var iteration in Benchmark.Iterations)
+            {
```

Comment:
FYI, this filename has a typo:
BrotliPerfomanceTests.cs => BrotliPerformanceTests.cs